app/sessionStore.js

```diff
@@ -412,6 +412,7 @@ module.exports.defaultAppState = () => {
         guid: [],
         timestamp: 0
       }
-    }
+    },
+    menubar: {}
```

Comment:
shouldn't this be stored in windowState instead? Wouldn't this mirror to all open windows?